cysync/src/pages/mainApp/sidebar/settings/tabViews/deviceHealth/deviceAuth.tsx

```diff
@@ -358,6 +352,10 @@ const DeviceAuth: React.FC<DeviceSettingItemProps> = ({
     }
   };
 
+  if (isRefresh) {
+    handleRetry();
+  }
+
```

Comment:
This can run multiple times. Put this code inside a useEffect and in the dependencies add `location.search`

Reply:
@irshadCypherock please recheck, I've moved it back inside the useEffect that runs only once.